apps/ledger-live-desktop/src/newArch/features/ModularDrawer/screens/AssetSelection/index.tsx

```diff
@@ -58,8 +58,8 @@ const AssetSelection = ({
         />
       )}
       <SearchInputContainer />
-      {error && refetch ? (
-        <GenericError onClick={refetch} />
+      {error ? (
+        <GenericError onClick={refetch || (() => {})} />
```

Comment:
onClick is optional, i'm not sure the refetch button should appear when it doesn't do anything? 

```suggestion
        <GenericError onClick={refetch} />
```